src/js/adjustHeights.js

```diff
@@ -1,27 +1,42 @@
 import $ from 'jquery';
 import _ from 'lodash';
 
-const setHeightsArray = [
-  // { className: 'flex-col--2__h1', parentId: 'naprawa'},
-  // { className: 'flex-col--2__p--1', parentId: 'naprawa'},
-  // { className: 'flex-col--2__p--2', parentId: 'naprawa'},
-  // { className: 'flex-col--2__list', parentId: 'naprawa'},
+const setHeightsArrayDesktop = [
+  { className: 'section__header', parentId: 'naprawa' },
+  { className: 'section__p--1', parentId: 'naprawa' },
+  { className: 'section__p--2', parentId: 'naprawa' },
+  { className: 'section__list-div', parentId: 'naprawa' },
+  { className: 'section__header', parentId: 'produkcja' },
+  { className: 'section__p--1', parentId: 'produkcja' },
+  { className: 'section__p--2', parentId: 'produkcja' },
+  { className: 'section__list-div', parentId: 'produkcja' }
 ];
 
+const setHeightsArrayAll = [
+  { className: 'section__p--3', parentId: 'produkcja' }
+];
 
 // adjust height of a div to the highest child of all considered divs
-export function setHeights() {
-  setHeightsArray.forEach((arrayEl) => {
-    const elements = $(`#${arrayEl.parentId}`).find(`.${arrayEl.className}`).toArray();
-    const heights = elements.map(it => it.scrollHeight);
-    const maxHeight = _.max(heights);
-    elements.forEach((el) => { el.style.height = `${maxHeight}px`; });
-  });
+function setHeightsInner(array) {
```

Comment:
rename `array` to `elementsWithParents`
and `arrayEl` to `elementWithParent`

- [ ] rename `array`